.github/workflows/release.yml

```diff
@@ -34,7 +34,6 @@ jobs:
         runs-on: ${{ matrix.os }}
         permissions:
             id-token: write # required for publishing to npm with provenance
-            contents: write # so we can deploy to GitHub pages
```

Comment:
Since we don't publish to GH in this workflow anymore, no need for this extra permission.